cypress/e2e/4-extensions.js

```diff
@@ -0,0 +1,76 @@
+import { Extension } from '@colony/colony-js';
+
+const testExtensionManagementFlow = extensionId => {
+  // Can install extension
+  cy.getBySel('install-extension-button').click().wait(5000);
+  cy.getBySel('extension-status-tag').should(
+    'have.text',
+    'Disabled',
+  );
+
+  // Can enable extension
+  cy.getBySel('close-gas-station-button').click();
+  cy.getBySel('enable-extension-button').click();
+  if (extensionId === Extension.Whitelist) {
+    cy.getBySel('policy-selector').eq(1).click({ force: true });
+  }
+  cy.getBySel('setup-extension-confirm-button').click().wait(5000);;
+  cy.getBySel('extension-status-tag').should(
```

Comment:
For all the waits you pointed out, it would not be possible to use timeout as by the way the selector is implemented there's **always** going to be an `extension-status-tag` available for selecting, so if we don't use wait then we risk grabbing the wrong one because the UI will not update fast enough.

With that being said, I have an idea that could possibly allow the use of timeouts instead of wait by changing the selectors, so I'll try that